### PR TITLE
Waiting $pidfile instead of $pid

### PR DESCRIPTION
**Pull request by RyoNkmr**
I found it seems to timeout very often with a message `timeout value 10000` and
it will be failed to wait for the process.

> wait pid_file, wait --pid pid
>
> https://www.rabbitmq.com/rabbitmqctl.8.html#wait

Thank you for the helpful images.

### CODE REVIEW

**Comment by w1ck3dg0ph3r** [comment]
Thanks!
Is this timeout caused by rabbitmq actually taking longer than default 10s to start? Because in that case I don't see how waiting for pidfile is different. Maybe I'm missing something?

**Comment by RyoNkmr** [comment]
@w1ck3dg0ph3r 

> Is this timeout caused by rabbitmq actually taking longer than default 10s to start?

`rabbitmqctl wait` is not for pid, but for the path where pid file is.
then `rabbitmqctl wait $pid` does just stop til 10s anyway whether the process gets succeeded or failed. because the code with `cat` gets pid itself. not its path.

**Comment by w1ck3dg0ph3r** [comment]
Oh, thanks, I see I've missed `--pid` argument. Would it be better to set RABBITMQ_PID_FILE to static path without instance id in it, and just wait for that path? Thus getting rid of the loop in the script?

**Comment by RyoNkmr** [comment]
> Would it be better to set RABBITMQ_PID_FILE to static path ... Thus getting rid of the loop in the script?

You're completely right, the loop is removed from the script!(in c83f082)

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,9 @@ FROM rabbitmq:alpine
 
 ENV RABBIT_USER=taiga \
 		RABBIT_PASSWORD=qwerty \
-		RABBIT_VHOST=taiga
+		RABBIT_VHOST=taiga \
+    RABBITMQ_ERLANG_COOKIE=taiga \
```
Comment (RyoNkmr) [inline Dockerfile:6]: `RABBITMQ_ERLANG_COOKIE` seems to be required in the latest image

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,9 @@ FROM rabbitmq:alpine
...
+    RABBITMQ_PID_FILE=$RABBITMQ_DATA_DIR/rabbitmq.pid
```
Comment (RyoNkmr) [inline Dockerfile:7]: > Environment Variables
> `RABBITMQ_PID_FILE` : File in which the process id is placed for use by rabbitmqctl wait.
> 
> https://www.rabbitmq.com/relocate.html#environment-variables

**Review by w1ck3dg0ph3r — APPROVED** [review]
Thanks! Looks good.

**Pull request merged**